src/site/xdoc/metrics.xml

```diff
@@ -24,6 +24,7 @@ under the License.
     <title>
       Apache HBase (TM) Metrics
     </title>
+    <link rel="shortcut icon" href="/images/favicon.ico" />
```

Comment:
Is there a way to put this in a common header so it's managed in one place?

Reply:
Good idea, I'll look into this.